code/modules/antagonists/clockcult/clock_items/clock_weapons/ratvarian_shield.dm

```diff
@@ -0,0 +1,84 @@
+//Subtype of (riot) shield because of already implemented shieldbash stuff aswell as integrity and simillar things
+//ratvarian shield: A shield that absorbs energy from attacks and uses it to empower its bashes with remendous force. It is also quite resistant to damage, though less so against lasers and energy weaponry.
+
+/obj/item/shield/riot/ratvarian
+	name = "ratvarian shield"
+	icon_state = "ratvarian_shield" //Its icons are in the same place the normal shields are in
+	item_state = "ratvarian_shield"
+	desc = "A resilient shield made out of brass.. It feels warm to the touch."
+	var/clockwork_desc = "A powerful shield of ratvarian making. It absorbs blocked attacks to charge devastating bashes."
+	armor = list("melee" = 80, "bullet" = 70, "laser" = -10, "energy" = -20, "bomb" = 60, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
+	shield_flags = SHIELD_FLAGS_DEFAULT
+	max_integrity = 300 //High integrity, extremely strong against melee / bullets, but still quite easy to destroy with lasers and energy
+	repair_material = /obj/item/stack/tile/brass
+	var/dam_absorbed = 0
+	var/bash_mult_steps = 30
+	var/max_bash_mult = 4
+
+/obj/item/shield/riot/ratvarian/examine(mob/user)
+	if((is_servant_of_ratvar(user) || isobserver(user)))
+		desc = clockwork_desc
+		desc +="\n <span class='inathneq_small'>The shield has absorbed [dam_absorbed] damage, multiplying the effectiveness of its bashes by [calc_bash_mult()]</span>"
+	. = ..()
+	desc = initial(desc)
```

Comment:
```suggestion
	. = ..()
	if((is_servant_of_ratvar(user) || isobserver(user)))
		. += clockwork_desc
		. +="\n <span class='inathneq_small'>The shield has absorbed [dam_absorbed] damage, multiplying the effectiveness of its bashes by [calc_bash_mult()]</span>"

```

Reply:
Not feasible as it is supposed to only display clockwork desc if examined by a observer / clockie, not normal desc + clockwork desc.
Clockie / normal desc is also supposed to be above the other special stuff for examine (e.g. materials, if its damaged, etc) ..() does, with only the multiplier supposed to be below. This means desc needs to be edited, as it is used in ..()

The descriptions in comparison: 
With the change:

![](https://cdn.discordapp.com/attachments/580545840972562454/711650174216437760/unknown.png)

Without the change:

![](https://cdn.discordapp.com/attachments/580545840972562454/711654776311513098/unknown.png)

I can partially comply with this and attach the part about the multiplier via . +=, but desc itself requires the previous implementation.